src/vs/platform/extensionManagement/node/extensionManagementService.ts

```diff
@@ -148,7 +148,10 @@ export class ExtensionManagementService implements IExtensionManagementService {
 					const readme = children.filter(child => /^readme(\.txt|\.md|)$/i.test(child))[0];
 					const readmeUrl = readme ? URI.file(path.join(extensionPath, readme)).toString() : null;
 
-					const local: ILocalExtension = { id, manifest, metadata, path: extensionPath, readmeUrl };
+					const changelog = children.filter(child => /^changelog(\.txt|\.md|)$/i.test(child))[0];
+					const changelogUrl = changelog ? URI.file(path.join(extensionPath, changelog)).toString() : null;
```

Comment:
This code is a simple copy paste from the `readme` counterpart.
Do you think would it be worth to rewrite in a more coincise way like `['changelog', 'readme'].reduce(() => {...}`?